test/integration/localizations_test.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+require_relative "../test_helper"
+
+class LocalizationsTest < ActiveSupport::TestCase
+  LOCALIZATION_DIRECTORY = "data/localizations"
+  CATEGORY_DIRECTORY = "data/categories"
+  ATTRIUBTES_DIRECTORY = "data/attributes.yml"
+  VALUES_DIRECTORY = "data/values.yml"
+
+  Dir.glob("#{LOCALIZATION_DIRECTORY}/categories/*.yml").each do |file|
+    language = File.basename(file, ".yml")
+    localizations = YAML.load_file(file)
+    Dir.glob("#{CATEGORY_DIRECTORY}/*.yml").each do |file|
+      vertical = File.basename(file, ".yml")
+      categories = YAML.load_file(file)
+      test "#{vertical} has localization keys in #{language} for every category" do
```

Comment:
Can you add a newline between these? I almost missed that there was a test here since it's hidden amongst other code